documentation/api/io.strimzi.api.kafka.model.kafka.listener.GenericKafkaListener.adoc

```diff
@@ -185,8 +167,6 @@ A new loadbalancer service is created for every Kafka broker pod.
 An additional loadbalancer is created to serve as a Kafka _bootstrap_ address.
 Loadbalancers listen to the specified port number, which is port `9094` in the following example.
 +
-You can use the `loadBalancerSourceRanges` property to configure xref:property-listener-config-source-ranges-reference[source ranges] to restrict access to the specified IP addresses.
-+
```

Comment:
I would keep this -> I think it is good to link to these things to make users aware of them.